test/e2e/config/config.yaml

```diff
@@ -5,8 +5,8 @@ bridgeIP: 172.17.0.1
 operatorReplicas: 1
 artifactsDir: ../../_artifacts
 
-certManagerVersion: v1.9.2
+certManagerVersion: v1.11.1
 certManagerChartURL: https://charts.jetstack.io/charts/cert-manager-${CERT_MANAGER_VERSION}.tgz
 
-rancherVersion: 2.7.9
+rancherVersion: 2.8.2
```

Comment:
Are we using this version? I noticed there's a different one hardcoded when installing rancher.

Reply:
this is Rancher helm chart version, with this Helm chart we are deploying latest image `v2.9-head`